### PR TITLE
Mes ms 4783 catc1e fail journey

### PR DESCRIPTION
**Pull request by turnerc1**
## Description
Cat c1e failed journey
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Comment by turnerc1** [comment]
This is @JCarterDVSA story

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 1]
```diff
@@ -59,6 +59,9 @@ When('I fail the eye sight test', () => {
 });
 
 When('I complete the waiting room to car page with the following vehicle checks', function (table) {
+  console.log('table', table);
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:62]: dont need this, please remove

File: test/e2e/step-definitions/waitingroom-steps.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ Before({ tags: '@catc' }, () => {
   this.testCategory = 'c';
 });
 
-Before({ tags: '@catc1' }, () => {
+Before({ tags: '@catc1e' }, () => {
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroom-steps.ts:21]: we already have a c1e

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 3]
```diff
@@ -105,8 +109,13 @@ const standardUserJourney = (withDriverFault: boolean, manualTransmission: boole
 
 const showMeQuestions = (questions, questionResult) => {
   const showMeQuestionsArray = [questions, questionResult];
+  console.log('SHOWMEQUESTIONSARRAY: ', showMeQuestionsArray);
   const elements = element.all(by.id('vehicle-checks-question-selector'));
   elements.each((element, index) => {
+    console.log('------------------------------------------------------------');
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:115]: remove

File: test/e2e/step-definitions/waitingroomtocar-steps.ts [hunk 4]
```diff
@@ -69,6 +72,7 @@ const completeWaitingRoomPage = (questionResult, manualTransmission: boolean, te
   } else if (this.testCategory === 'c' || this.testCategory === 'c1') {
     multiShowAndTell(UI_TEST_DATA.testData.c, questionResult);
   } else if (this.testCategory === 'ce') {
+    console.log('UI TEST DATA ce: ', UI_TEST_DATA.testData.ce);
```
Comment (jono-basford-bjss) [inline test/e2e/step-definitions/waitingroomtocar-steps.ts:75]: please remove

**Review by jono-basford-bjss — CHANGES_REQUESTED** [review]

**Pull request closed without merging**